tests/testthat/helper-model.R

```diff
@@ -113,3 +132,15 @@ expect_accuracy <- function(pred, true, at_least) {
 expect_not_constant_predictions <- function(pred) {
   expect_true(length(unique(pred)) > 1)
 }
+
+expect_multi_predict_works <- function(model) {
+  adj <- parsnip::fit(model, mpg ~ ., data = mtcars)
+
+  expect_true(parsnip::has_multi_predict(adj))
+  expect_equal(parsnip::multi_predict_args(adj), "trees")
+
+  mp <- parsnip::multi_predict(adj, mtcars, trees = c(1, 3, 5))
+  expect_equal(nrow(mp), nrow(mtcars))
+  expect_equal(nrow(tidyr::unnest(mp, .pred)), 3*nrow(mtcars))
+  expect_error(multi_predict(adj, mtcars, trees = 6))
```

Comment:
why should this error? You can't use it for a single tree?

Reply:
because only 5 trees were built. The 6th doesn't exist. But I removed this test at multi_predict_lightgbm branch.